wasm-text.md

```diff
@@ -1045,6 +1225,74 @@ The following are helper functions for gathering and updating context.
     rule #idcFuncsAux(.Defns, (func      _:FuncSpec) FS => FS, IDX => IDX +Int 1, _ACC)
     rule #idcFuncsAux(.Defns, .Defns, _, ACC) => ACC
 
+    syntax Map ::= #idcTables    (Defns, Defns ) [function]
```

Comment:
Aren't these functions total?

The same question for the similar functions below.

Reply:
Some functions in this module are non-total because when doing text-to-abstract conversion they perform some validations. For example `#idcTables` checks that the table IDs are unique. If there are duplicate IDs, the result is undefined.